app/src/main/res/layout/list_prescribeddrugs_custom_drug.xml

```diff
@@ -9,7 +9,8 @@
   android:focusable="true"
   android:foreground="?selectableItemBackground"
   android:orientation="vertical"
-  app:cardCornerRadius="0dp">
+  app:cardCornerRadius="0dp"
+  app:cardElevation="0dp">
```

Comment:
Do we need card elevation if we're setting it as 0dp?

Reply:
Yes, by default card elevation will be 4dp. So we are specifically setting it to 0dp. I think at this point we can just make this a `FrameLayout` since we have a wrapper card view around the recyclerview.

Reply:
I have already done it in this commit. https://github.com/simpledotorg/simple-android/pull/2380/commits/f002030a472f4be3ad9d1615451cc3ee9ca34a17